newsletter_automation/conf/apikey_conf.py

```diff
@@ -0,0 +1,4 @@
+"""
```

Comment:
This file should not be part of version control. Add it to .gitignore and then put instructions in the README as setup instructions.